app/models/pic.rb

```diff
@@ -33,6 +33,16 @@ class Pic < ActiveRecord::Base
       "#{basename(attachment, style)}_#{style}"
     end
   end
+
+  # Returns true when this represents a usable picture.
+  def image_present?
+    pic_file_name.present? && pic_content_type.start_with?('image/')
```

Comment:
Some more context around why this code will be needed might be nice. I'm guessing it's shimming for something coming?

Reply:
There was some defensive programming in the code `pic&.pic&.url` I believe. I was trying to replicate why this might be the case. I don't know why Paperclip would return `nil` in the second case. I could just change this to `pic.nil?`?

Reply:
Gotcha, was just wondering why the content type was being checked. We'll be ok on production, it seems:

```
irb(main):005:0> Pic.where('pic_content_type not like "image/%"').count
=> 0
```